### DISCUSSION TITLE
I can't change my profile picture

### DISCUSSION THREAD

**Discussion by TheMarci** [community/community#60695] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I just want to change my profile picture but can't. No matter what image I try I always got the same error message. 
![err](https://github.com/community/community/assets/33132540/ebfb5ed1-fb45-4758-9c58-adc157dbb983)

**Comment by TheMarci** [comment]
![what](https://github.com/community/community/assets/33132540/845caefb-62a3-4715-b1d8-b96cd9776bd0)
Also just noticed by topic it shows the image I wanted to change it to but in the right upper corner its still the old and if I click on my profile it still shows the old.

**Comment by melvincwng** [comment, marked as answer]
Hey @TheMarci, ensure your new profile picture is in either one of the following formats:

1. Png
2. Jpg
3. Gif

It must also be less than 1MB in size and less than 3000 pixels by 3000 pixels.

Once re-uploaded, it takes a while (usually a few minutes) before the new profile picture is in sync across the GitHub platform

For more info, you can refer to https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile

Hope this answers your question 😃

**Reply by TheMarci** [reply]
Thanks for your help. I read the docs and resized the image to 500x500 and it worked.

**Reply by melvincwng** [reply]
Always a pleasure to help 😀

**Comment by artemismoon1989** [comment]
of course, I did as written in this area.  but there is nothing change.

**Comment by teddis** [comment]
I cannot change my avatar. 👎  It's 1251x1251, JPG, 307KB
![image](https://github.com/community/community/assets/1139314/9f1f3810-12a2-40a7-9704-5f536c6f83cd)

**Comment by ubidev** [comment]
Hi
currently trying with an ultra simple 20kb 330x275 logo either png or jpg Iget an endless processing loop on chrome or edge...
![image](https://github.com/user-attachments/assets/e1c3fab8-0acb-4337-9aaf-873abe317ccc)

What can I do to assist resolving this annoyance.
Thanks in advance for your time
